src/usefulLib.hpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <vector>
+char to(int p)
+{
+    return ((p < 10) ? (p + '0') : (p - 'A' - 10));
```

Comment:
скобки избыточны ииии.... 10? что здесь делает 10? Магические числа не стоит использовать. Выход - использование константы или альтернативное решение.
```suggestion
    const int count_numbers = 10;
    return p < count_numbers  ? p + '0' : p - 'A' - count_numbers ;
```